backend/backend/api/routes.py

```diff
@@ -130,12 +130,12 @@ async def read_root():
     </style>
```

Comment:
The code patch introduces a new button for the Research assistant feature, but it seems to reuse the same input field id for both the Test and Research buttons. This could lead to unexpected behavior when switching between these features. Consider using unique ids for each form element.